features/step_definitions/api/plan_features_steps.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+When('I {enable} the feature {string}') do |enable, feature_name|
+  within(:xpath, "//tr[td[text() = '#{feature_name}' and @class='title']]") do
+    if enable
+      find('i.excluded').click
+    else
+      find('i.included').click
+    end
+  end
+end
+
+When('I click {string} for the feature {string}') do |action, feature_name|
```

Comment:
Do you think this step would be useful for anything but plans? Maybe this would be a little better?

```suggestion
When('I click {string} for the plan feature {string}') do |action, feature_name|
```

Same question applies to all new steps here.

Reply:
I think features only apply to plans, so yeah, feature == plan feature.
No problem, we can add this to make it less confusing.

Reply:
Done